app/models/ems_refresh.rb

```diff
@@ -171,7 +171,13 @@ def self.queue_merge(targets, ems, create_task = false)
 
     # Items will be naturally serialized since there is a dedicated worker.
     MiqQueue.put_or_update(queue_options) do |msg, item|
-      targets = msg.nil? ? targets : (msg.args[0] | targets)
+      # Handle merging targets for old queue items that still have in args[0]
+      if msg && msg.args[0]
+        msg.data.concat(msg.args.shift)
+        msg.args = []
+      end
```

Comment:
@Fryguy made some changes so my comment was removed but same question here ^^ do you think this is needed?  My worry is if we don't clear args then we'll get refresh delivered with `EmsRefresh.refresh(args, data)`

Reply:
I'm ok with this for now, but ultimately, I don't think it's needed, or we should deal with it in a data migration.  I don't like live "fixing" of data.

Reply:
Okay I'll work on a data migration so we can get rid of this, agree we don't want to carry this forever

Reply:
Can be removed once https://github.com/ManageIQ/manageiq-schema/pull/107 is in